pyignite/connection/connection.py

```diff
@@ -28,6 +29,8 @@
 
 CLIENT_STATUS_AUTH_FAILURE = 2000
 
+logger = logging.getLogger('.'.join(__name__.split('.')[:-1]))
```

Comment:
Is it a good idea to make it global?

Reply:
Yep, it is common approach.
For example, [see aiohttp](https://github.com/aio-libs/aiohttp/blob/09ac1cbb4c07b4b3e1872374808512c9bc52f2ab/aiohttp/log.py)